internal/client/client_test.go

```diff
@@ -74,103 +58,57 @@ func TestAPIResponse(t *testing.T) {
 func TestGetRaw(t *testing.T) {
 	// Get raw data
 	b, resp, err := GetRaw(testState.socket, "GET", "/api/status")
-	if err != nil {
-		t.Fatalf("GetRaw failed with a client error: %v", err)
-	}
-	if resp != nil {
-		t.Fatalf("GetRaw request failed: %v", err)
-	}
-	if len(b) == 0 {
-		t.Fatal("GetRaw returned an empty string")
-	}
+	require.NoError(t, err)
+	require.Nil(t, resp)
+	require.Greater(t, len(b), 0)
+
 	// Get an API error
 	b, resp, err = GetRaw(testState.socket, "GET", "/api/v0/blueprints/list?offset=bad")
-	if err != nil {
-		t.Fatalf("GetRaw bad request failed with a client error: %v", err)
-	}
-	if resp == nil {
-		t.Fatalf("GetRaw bad request did not return an error: %v", b)
-	}
-	if resp.Status != false {
-		t.Fatalf("Status != false: %#v", resp)
-	}
-	if len(resp.AllErrors()) < 1 {
-		t.Fatalf("GetRaw error did not return error message: %#v", resp)
-	} else if resp.Errors[0].ID != "BadLimitOrOffset" {
-		t.Fatalf("GetRaw error ID is not BadLimitOrOffset: %#v", resp)
-	}
+	require.NoError(t, err)
+	require.NotNil(t, resp)
+	require.Equal(t, false, resp.Status)
```

Comment:
Why not to use https://godoc.org/github.com/stretchr/testify/require#False here?

Reply:
updated